frontend/app/src/components/asset-manager/NewlyDetectedAssetTable.vue

```diff
@@ -160,7 +160,7 @@ const ignoreTokens = async () => {
       </template>
 
       <template #item.address="{ item }">
-        <hash-link :text="item.address" />
+        <hash-link :text="item.address" no-link />
```

Comment:
I made it no-link for starters because in order for the link to work fine we need to also get the chain, which is not impossible but it would be a bit more complicated.